src/molecules/Content/__tests__/Content.test.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import { Image } from 'apm-mimas';
+import { image } from '../../../__testdata__/image';
+import Content from '../Content';
+
+// automatically unmount and cleanup DOM after the test is finished
+afterEach(cleanup);
+
+const apmImage = <Image image={image} />;
+
+const defaultProps = {
+  title: 'this is a title',
+  bodyHtml: 'body field'
+};
+
+test('renders ContentHeader with correct classes and content', () => {
+  const { getByText } = render(<Content title={defaultProps.title} />);
+
+  const node = getByText('this is a title');
+
+  expect(node.classList.contains('hdg')).toBe(true);
+  expect(node.classList.contains('hdg-1')).toBe(true);
+});
+
+test('renders ContentBody if bodyHtml prop exists', () => {
+  const { getByText } = render(
+    <Content title={defaultProps.title} bodyHtml={defaultProps.bodyHtml} />
+  );
+
+  const node = getByText('body field');
+
+  expect(node.classList.contains('userContent')).toBe(true);
```

Comment:
This test syas its going to see if the body is rendered,  but instead its checking the className seems like it should either check the actual output or be re-titled to say its checking the className

Reply:
This might seem really nit pickiy  but I think it would make the test clearer to read